test/e2e/puppeteer.js

```diff
@@ -46,13 +46,16 @@ const exceptionList = [
 
 /* CONFIG VARIABLES END */
 
-const LAST_REVISION_URLS = {
-	linux: 'https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/LAST_CHANGE',
-	mac: 'https://storage.googleapis.com/chromium-browser-snapshots/Mac/LAST_CHANGE',
-	mac_arm: 'https://storage.googleapis.com/chromium-browser-snapshots/Mac_Arm/LAST_CHANGE',
-	win32: 'https://storage.googleapis.com/chromium-browser-snapshots/Win/LAST_CHANGE',
-	win64: 'https://storage.googleapis.com/chromium-browser-snapshots/Win_x64/LAST_CHANGE'
+const PLATFORMS = {
+	linux: 'linux',
+	mac: 'mac',
+	mac_arm: 'mac_am64',
+	win32: 'win',
+	win64: 'win64'
 };
+const OMAHA_PROXY = 'https://omahaproxy.appspot.com/all.json';
+
+const chromiumChannel = 'dev'; // stable -> beta -> dev -> canary (Mac and Windows) -> canary_asan (Windows)
```

Comment:
@mrdoob @epreston This option is configurable -- it can be configured to be latest stable, beta, or dev version.

Reply:
From https://www.chromium.org/getting-involved/dev-channel/:
> The release channels for chrome range from the most stable and tested (Stable channel) to completely untested and likely least stable (Canary channel). You can run all channels alongside all others, as they do not share profiles with one another. This allows you to play with our latest code, while still keeping a tested version of Chrome around.

> * Stable channel: This channel has gotten the full testing and blessing of the Chrome test team, and is the best bet to avoid crashes and other issues. It's updated roughly every two-three weeks for minor releases, and every 6 weeks for major releases.
> * Beta channel: If you are interested in seeing what's next, with minimal risk, Beta channel is the place to be. It's updated every week roughly, with major updates coming every six weeks, more than a month before the Stable channel will get them.
> * Dev channel: If you want to see what's happening quickly, then you want the Dev channel. The Dev channel gets updated once or twice weekly, and it shows what we're working on right now. There's no lag between major versions, whatever code we've got, you will get. While this build does get tested, it is still subject to bugs, as we want people to see what's new as soon as possible.
> * Canary build: Canary builds are the bleeding edge. Released daily, this build has not been tested or used, it's released as soon as it's built.
> * Other builds: If you're extra brave, you can download the latest working (and that's a very loose definition of working) build from [download-chromium.appspot.com](https://download-chromium.appspot.com/). You can also look for a more specific recent build by going to [the Chromium continuous build waterfall](http://build.chromium.org/), looking at the number near the top under "LKGR", and then going to [this Google Storage bucket](http://commondatastorage.googleapis.com/chromium-browser-continuous/index.html) and downloading the corresponding build.
Note: Early access releases (Canary builds and Dev and Beta channels) will be only partly translated into languages other than English. Text related to new features may not get translated into all languages until the feature is released in the Stable channel.

Reply:
That looks like its for people doing developing for chromium.  Not people wanting to use the latest release chromium.

Unless there's some goal that requires it. Please consider configuring it to use latest release instead of Canary.  It might be a lot of work debugging mystery chromium beta builds that break and fix them selves 3 times a day.

Reply:
Ok, I will change it to `stable`.